data_highlight/highlighter.py

```diff
@@ -105,3 +92,67 @@ def export(self, filename: str):
             fOut.write("</span>")
 
         fOut.close()
+
+    def limitedLines(self):
+        """
+            If  numberOfLines were limited
+        :return:
+        """
+
+        with open(self.filename, 'r') as file:
+            sampleLines = file.read()
+
+        lineCtr = 0
+        lines = []
+        strLines = sampleLines.splitlines()
+        strLines= strLines[0:self.numberOfLines]
+        stringToChar = '\n'.join(str(e) for e in strLines)

```

Comment:
would you mind adding a comment, just to explain what we're doing with `stringToChar` above.  Hold on - are we separating the lines with `newline` chars?